week-2/C-comparison-operators/exercise.js

```diff
@@ -7,16 +7,16 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = enoughSpaceInRoom === roomMaxCapacity; // finish this statement
```

Comment:
I don't think this is quite right. The exercise asks you to check if `studentCount` plus `mentorCount` is greater than the `roomMaxCapacity`. How could you fix that?